blint/sbom.py

```diff
@@ -193,8 +192,7 @@ def create_sbom(
         return True
     else:
         # Write the JSON output directly to stdout
-        sys.stdout.write(sbom_content)
-        LOG.debug(f"SBOM json output printed successfully to stdout")
+        LOG.info(sbom_content)
```

Comment:
Interesting. I think we need `sys.stdout.write` as per the last commit since rich logging might add timestamp, highlights, and emojis etc. My last comment was about disabling the logging completely. Let me work on this feature differently and share a PR.

Reply:
sorry, I was trying something out (forgot to create a new branch) and thought it would be a good idea to make use of logger and support raw console output. 
The moment you can share your changes, I'll delete this PR and create a new one - removing my tests.